package_builder/requirements.txt

```diff
@@ -18,7 +18,7 @@ epicscorelibs
 ess-streaming-data-types
 funcsigs
 future
-genie_python[plot]  # Ensure we retain the [plot] extras (for matplotlib plotting)
+genie_python[plot] @ git+https://github.com/ISISComputingGroup/genie@Ticket8597  # Ensure we retain the [plot] extras (for matplotlib plotting)
```

Comment:
Merge `genie_python` pr first, make a pypi release, then put this back, then merge this PR.